test/functional/testlib.bash

```diff
@@ -1161,6 +1160,42 @@ install_bundle() {
 
 }
 
+# Adds the specified file to the zero or delta pack for the bundle
+# Parameters:
+# - BUNDLE: the name of the bundle
+# - ITEM: the file or directory to be added into the pack
+# - FROM_VERSION: the from version for the pack, if not specified a zero pack is asumed
+add_to_pack() {
```

Comment:
Add this to the readme?

Reply:
The way I see it, there are two types of functions in the library, those functions that are really of not use to the end user (from a test script) but are useful for other library functions (kind of like a private function), and those that are useful for users writing tests. In this case I don't see much value for a person using this function in isolation, so I'm not sure how important is to add this function to the readme.